collectors/deployments_collector.go

```diff
@@ -51,6 +52,21 @@ func NewDeploymentsCollector(
 		[]string{"bosh_deployment", "bosh_stemcell_name", "bosh_stemcell_version", "bosh_stemcell_os_name"},
 	)
 
+	deploymentVMInfoMetric := prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: namespace,
+			Subsystem: "deployment",
+			Name:      "vm_type_counter",
```

Comment:
The name says that this metric is a `counter` but it's defined as a `gauge`

Reply:
I renamed it to instance_count as instance is a the field name of the DeploymentInfo structure. vm_type is anyway a lable so it should not be in the name.

However I left the variable type as gauge because prometheus documentation states that a counter should be cumulative.